package/Makefile.winx86

```diff
@@ -7,19 +7,19 @@ NSIS_OPTS     := /V3
 NSIS_DIR      := $(ROOT_DIR)/package/winx86
 NSIS_SCRIPT   := $(NSIS_DIR)/dronin.nsi
 NSIS_TEMPLATE := $(NSIS_DIR)/dronin.tpl
-NSIS_HEADER   := $(BUILD_DIR)/ground/gcs/dronin.nsh
+NSIS_HEADER   := $(PACKAGE_DIR)/$(GCS_PACKAGE_NAME)/dronin.nsh
 
-standalone: 
+standalone: gcs
```

Comment:
Yeah, I think adding a target that completes all necessary steps would be worthwhile, even if Jenkins doesn't use it.